tests/test_utils.py

```diff
@@ -59,6 +59,12 @@ def __eq__(self, other: Any) -> Any:
         return False
 
 
+class Baz(Foo):
+    @classmethod
```

Comment:
can you test it on `a @staticmethod ` as well?

Reply:
done